### PR TITLE
Added documentation to buffers, policies and values

### PR DESCRIPTION
**Pull request by Zeus3101**
I followed the PyTorch syntax for documentation, except for the `:param var: (type) description` notation. This looks good when I scroll over on VS Code. Solves part of #104

### CODE REVIEW

**Comment by Sharad24** [comment]
Have you tried building the documentation locally? If yes, can you post a picture of how it looks?

**Comment by Zeus3101** [comment]
image

This is the code for the docstring. It looks like this when you hover over the definition:
image

**Comment by Sharad24** [comment]
Hmm, not this. Run `sphinx-versioning --use-master-conf --use-master-templates build --greatest-tag --whitelist-branches master docs/source docs/build/html` from the root directory of the repo and then `open docs/build/html/index.html`.

**Comment by Sharad24** [comment]
For example this is what I see for DDPG

Screenshot 2020-05-08 at 1 37 31 AM

**Comment by ajaysub110** [comment]
> Hmm, not this. Run `sphinx-versioning --use-master-conf --use-master-templates build --greatest-tag --whitelist-branches master docs/source docs/build/html` from the root directory of the repo and then `open docs/build/html/index.html`. 

You can also just run `make html` from the directory with Makefile

**Comment by Sharad24** [comment]
`make html` would not collect the docstrings though, no?

**Comment by ajaysub110** [comment]
It works

**Comment by Zeus3101** [comment]
Let me know if I need to make changes in `logger.py` and `noise.py` as I wasn't exactly sure about some of the types and descriptions @Sharad24 @ajaysub110 @Het-Shah

**Comment by Sharad24** [comment]
Fix Travis first?

**Comment by Sharad24** [comment]
Travis fails?

**Comment by Sharad24** [comment]
?

**Comment by Zeus3101** [comment]
Not really sure why there are build errors when I've only done reformatting and added documentation. Will sort these out tomorrow.

**Comment by Sharad24** [comment]
Hmm, even #116 is failing. I'll try to rerun master build once to see if there are errors there.

**Comment by Sharad24** [comment]
FYI @Zeus3101 , #116 is building now

File: genrl/deep/common/base.py [hunk 1]
```diff
@@ -54,29 +92,63 @@ def get_action(self, state, deterministic=False):
...
+    Basic implementation of a general Actor Critic
+    """
     def __init__(self):
         super(BaseActorCritic, self).__init__()
 
         self.actor = None
         self.critic = None
 
     def get_action(self, state, deterministic=False):
-        state = torch.as_tensor(state).float()
```
Comment (Het-Shah) [inline genrl/deep/common/base.py:77]: Are you sure omitting this line is not having any effect to the agents?

File: genrl/deep/common/base.py [hunk 2]
```diff
@@ -54,29 +92,63 @@ def get_action(self, state, deterministic=False):
...
+        :param deterministic: True if the action space is deterministic, \
+else False
+        :type state: Tensor
+        :type deterministic: boolean
+        :returns: action
+        """
         return self.actor.get_action(state, deterministic=deterministic)
 
     def get_value(self, state):
-        state = torch.as_tensor(state).float()
```
Comment (Het-Shah) [inline genrl/deep/common/base.py:81]: Same as before

File: genrl/deep/common/base.py (hunk 1 above)
Comment (Zeus3101) [reply genrl/deep/common/base.py]: This may be the issue. Thanks for pointing it out!

**Comment by Zeus3101** [comment]
The build errors don't show up locally. I've run the exact same tests and they work fine. Not sure why Travis is acting up.

**Comment by Zeus3101** [comment]
Abandoned in favour of #126

**Pull request closed without merging**